src/libraries/System.DirectoryServices.AccountManagement/tests/PrincipalTest.cs

```diff
@@ -15,9 +15,8 @@ public abstract class PrincipalTest : IDisposable
 
         private void RefreshContext()
         {
-            // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Unit test password.")]
-            string username = "Administrator", password = "Adrumble@6";
-
+            string username = "Administrator";
+            string password = Environment.GetEnvironmentVariable("TESTPASSWORD");
```

Comment:
These tests don't have a `[Fact]` attribute so they're not even trying to run. But I haven't yet examined whether we should try to run them, so I'm not deleting them. When we do, I believe the best practice for simple creds is to use an environment variable.